public/data/GautamHegde.json

```
@@ -0,0 +1,27 @@
+{
+    "name": "Gautam Hegde",
+    "type": "personal",
+    "bio": "Front end developer | Open Source Enthusiast!",
+    "avatar": "https://github.com/Gautam-Hegde.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/Gautam-Hegde",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/GautamHegde12",
+        "icon": "twitter"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Started Web Development",
+        "date": "June 2019",
+        "icon": "dollar",
+        "color": "grey",
+        "description": "Started Learning Web Development"
+      }
+    ]
+  }
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

---

@Gautam-Hegde Thanks for adding this, please mark this as resolved.